csrankings-j.csv

```diff
@@ -784,8 +784,9 @@ Jens-Peter Redlich,Humboldt University of Berlin,http://sar.informatik.hu-berlin
 Jeong hwan Jeon,UNIST,https://rml.unist.ac.kr,ZTe13LgAAAAJ
 Jeong-Gil Ko,Yonsei University,http://www.eis-lab.org,5zP95ZQAAAAJ
 Jeong-Hyon Hwang,University at Albany - SUNY,https://www.albany.edu/ceas/jeong-hyon-hwang.php,uurRDOwAAAAJ
+Jeonggeun Kim,Kyungpook National University,https://sites.google.com/view/knunextcom/,lvU4X6IAAAAJ

```

Comment:
Does not match DBLP name: https://dblp.org/pid/20/507.html

Reply:
The URL is not about our faculty.

This is our faculty: https://dblp.org/pid/150/9685.html

And I have fixed the name accordingly.